packages/vue-client/src/assets/main.css

```diff
@@ -44,3 +44,8 @@ a,
   display: flex;
   flex-direction: column;
 }
+
+.labelledContainer {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
This seems odd to me. What does a container being *labelled* have to do with it having a flex layout applied?

Reply:
Mainly that I believe this is useful for building the UI. Whenever there's a label to an input, we can wrap them in such a div with this class to display the label on top.

Reply:
You are right though, it can have a generally applicable name -> renamed to col
Thanks!